### ISSUE TITLE
Add reverse oneway-arrows if cycling is allowed in both directions

### ISSUE THREAD

**Issue by pietervdvn** [openstreetmap/iD#9646]
### Description

For my work, we needed a patched iD where an indication was given if it is bidirectional for cyclists - similar to many cycling maps.

Is there any interest in having this upstreamed?

### Screenshots

![afbeelding](https://github.com/openstreetmap/iD/assets/1466478/4bdba65c-2cc2-477e-903b-5de4e332635d)

**Comment by tyrasd** [comment]
Yeah, it would be fine to have a visual indication that there are oneway exceptions for some modes of transportation (e.g. bikes, buses, emergency, etc.).


> Is there any interest in having this upstreamed?

If you have already code for this, please share it as a PR. 😊 [^1]

[^1]: It could be the case that you encounter some conflicts as some things have changed in the meantime in respect to how the oneway arrows are rendered, but even in that case: please share it anyway.

**Comment by pietervdvn** [comment]
Oooffff, I'm not sure I have the code still around. It's been over 2 years and this was for a previous job...

**Comment by tyrasd** [comment]
Oh, that's understandable. The code that renders the `oneway=reversible` should be more or less reusable for these cases. It's only a little tricky not to overlook any of the tags which should be considered for this.
